### PR TITLE
Add NewFeature class

### PR DESCRIPTION
**Pull request by timomitchel**
This looks good - please check

### CODE REVIEW

File: new_feature.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class NewFeature
+
```
Comment (aziobrow) [inline new_feature.rb]: should we initialize this with an argument

File: new_feature.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+end
```
Comment (aziobrow) [inline new_feature.rb:5]: add another comment

**Review by aziobrow — CHANGES_REQUESTED** [review]
See what you think of my comment about adding an initialize

File: new_feature.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class NewFeature
+  def initialize(argument)
+    @argument = argument
```
Comment (timomitchel) [inline new_feature.rb:3]: Added the argument

Comment (aziobrow) [reply new_feature.rb:3]: looks great!

**Review by aziobrow — APPROVED** [review]

**Pull request merged**